pkgs/by-name/fl/flint/package.nix

```diff
@@ -2,34 +2,46 @@
   lib,
   stdenv,
   fetchurl,
-  gmp,
-  mpfr,
-  ntl,
+  fetchpatch,
   windows,
   autoconf,
   automake,
   gettext,
   libtool,
-  openblas ? null,
+  gmp,
+  mpfr,
+  ntl,
   blas,
   lapack,
+  boehmgc,
+  openblas ? null,
   withBlas ? true,
   withNtl ? !ntl.meta.broken,
+  withGc ? false,
 }:
 
 assert
   withBlas
   -> openblas != null && blas.implementation == "openblas" && lapack.implementation == "openblas";
 
-stdenv.mkDerivation rec {
-  pname = "flint3";
+stdenv.mkDerivation (finalAttrs: {
+  pname = "flint";
   version = "3.3.1";
 
   src = fetchurl {
-    url = "https://flintlib.org/download/flint-${version}.tar.gz";
+    url = "https://flintlib.org/download/flint-${finalAttrs.version}.tar.gz";
     hash = "sha256-ZNcOUTB2z6lx4EELWMHaXTURKRPppWtE4saBtFnT6vs=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-checkPhase-linker-errors";
+      url = "https://patch-diff.githubusercontent.com/raw/flintlib/flint/pull/2411.patch";
```

Comment:
Looks like the file may change after rebase. Maybe we should put the patch in Nixpkgs?

Reply:
Makes sense